### ISSUE TITLE
Rescript compile errors not being displayed on code.

### ISSUE THREAD

**Issue by henriquezago** [giraud/reasonml-idea-plugin#384]
plugin version: 0.106.1-2022.1

### Description
I know the README says `Rescript users, please use other editor.` so I come with the intention to start contributing here if possible.

If anyone could give me a hint on how to make the rescript compile errors display at the code where it origins, that would be amazing.
More importantly, is it possible to do so? Would be good to have an insight before setting up my dev env.
![Screen Shot 2022-07-23 at 11 00 51](https://user-images.githubusercontent.com/5287289/180608356-851f922a-0923-4f01-b52e-aaa989140b46.png)

I prefer Webstorm features to run tests, scripts and merge conflicts, but having to chase compile errors from the console is a bit counter-productive, so I keep using VSC as my main editor.

**Comment by giraud** [comment]
Hi. it should be able to display the error.
given the names of your component, do you use automatic namespacing in your project ?
is the project public and available ?

**Comment by henriquezago** [comment]
@giraud unfortunately the project is not public.
Forgive me, but I'm not entirely sure about the automatic namespacing, the project has many modules and their `bsconfig.json` have:
`name: "@[companyName]/[moduleName]"`
`namespace: true`

**Comment by henriquezago** [comment]
I've added a `System.out.println("line: " + line);` at `RescriptOutputAnalyzer.java:28` and the output was:
```
line: sh: -c: line 0: unexpected EOF while looking for matching `''
line: sh: -c: line 1: syntax error: unexpected end of file
line: File "/private/var/folders/gk/dz5wt7kj3gz03_xy6_d6fg0c0000gn/T/BS_walnut_monorepo/RangeCalendar.res", line 1, characters 0-0:
line: Error: Error while running external preprocessor
line: Command line: '/Users/henriquelobatozago/Development/walnut_monorepo/node_modules/bisect_ppx/ppx '/var/folders/gk/dz5wt7kj3gz03_xy6_d6fg0c0000gn/T/ppx47a6cdRangeCalendar.res' '/var/folders/gk/dz5wt7kj3gz03_xy6_d6fg0c0000gn/T/ppx7d57eaRangeCalendar.res'
line: 
2022-07-25 15:33:37,107 [  30433]   WARN - R.h.rincewind - Failure: Sys_error("/private/var/folders/gk/dz5wt7kj3gz03_xy6_d6fg0c0000gn/T/BS_walnut_monorepo/RangeCalendar.cmt: No such file or directory")
```

`RangeCalendar.res` is the file where I scrambled a module name to produce an error.

**Comment by giraud** [comment]
namespace: true is automatic namespacing. 
automatic namespacing is not really supported in plugin

**Comment by giraud** [comment]
if you want more debug, you can use the debug log settings in help | diagnostic tools.
use ReasonML for all debug are ReasonML.xxxx for a specific log

**Comment by henriquezago** [comment]
@giraud do you have any information or hints on supporting automatic namespacing?
It's been a while but I remember having no idea on how to proceed with this.

**Comment by giraud** [comment]
I suspect there is another problem that prevent error location to be parsed correctly for rescript logs. 
I have it sometimes even for projects that have no namespace. Some issue in console filters, I don’t know the root cause.

**Comment by giraud** [comment]
I found the root cause, it will be fixed in next release

**Comment by giraud** [comment]
fixed in 0.110
